lib/library.cpp

```diff
@@ -65,9 +66,12 @@ static void gettokenlistfromvalid(const std::string& valid, bool cpp, TokenList&
     }
 }
 
-Library::Error Library::load(const char exename[], const char path[])
+Library::Error Library::load(const char exename[], const char path[], bool verbose)
 {
+    // TODO: remove handling of multiple libraries at once?
     if (std::strchr(path,',') != nullptr) {
+        if (verbose)
+            std::cout << "handling multiple libraries '" + std::string(path) + "'" << std::endl;
```

Comment:
Wouldn't these messages be just as interesting for GUI users or users that run cppcheck in an IDE plugin.

Reply:
there are also plugins (i.e. vscode plugin) that will stop working if cppcheck outputs any unexpected info on stdout.

Reply:
do I understand it correctly that if using `--verbose` on the command line then cppcheck will start writing these messages? I like that it's possible to get this info so users can debug cppcheck analysis but I am skeptic about reusing that option.

Reply:
`--verbose` is essentially a debug option. That should not be used (and is currently not usable) for production.

Reply:
> --verbose is essentially a debug option. That should not be used (and is currently not usable) for production.

Imho this is not a debugging flag. It makes the warnings more verbose. I don't know why this would not be usable for production.

If you want to have some temporary output tweak.. can I suggest that you add a new flag that can later be removed when you are done.

Reply:
> Imho this is not a debugging flag. It makes the warnings more verbose. I don't know why this would not be usable for production.

You are totally right. I blanked on that.

> If you want to have some temporary output tweak.. can I suggest that you add a new flag that can later be removed when you are done.

This is not temporary but to provide the information which files are being used to debug potential issues. I will add an additional (for now undocumented) debug flag for this.